src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const headings = document.querySelectorAll('li');
```

Comment:
Using 'document.querySelectorAll('li')' selects all list items in the document. If the intention is to select only specific list items, consider using a more specific selector or ensuring the list items are within a specific parent element.